### PR TITLE
Feat/add webpay plus mall commit example

### PR DESCRIPTION
**Pull request by eugenioLopezRamos**
Add an example for committing a Webpay Plus Mall Transaction

### CODE REVIEW

File: app/controllers/webpay_controller.rb [hunk 1]
```diff
@@ -31,7 +31,43 @@ def commit
...
+    @resp = Transbank::Webpay::WebpayPlus::Transaction::refund(token: @token, amount: @amount)
+  end
 
+  def status
+    @req = params.as_json
+    @token = params[:token]
+    @resp = Transbank::Webpay::WebpayPlus::Transaction::status(token: @token)
   end
+
+  def mall_create
```
Comment (a1b4) [inline app/controllers/webpay_controller.rb]: Why this method is empty?

Comment (eugenioLopezRamos) [reply app/controllers/webpay_controller.rb]: It renders the 'mall_create' view

Comment (a1b4) [reply app/controllers/webpay_controller.rb]: you can write this as `def mall_create; end`

Comment (eugenioLopezRamos) [reply app/controllers/webpay_controller.rb]: I know, I just find that to be extremely ugly

Comment (eugenioLopezRamos) [reply app/controllers/webpay_controller.rb]: This style guide is against me, though
https://github.com/rubocop-hq/ruby-style-guide#no-single-line-methods

Comment (a1b4) [reply app/controllers/webpay_controller.rb]: 😂 it's ok

**Pull request merged**